src/LinuxTracing/SwitchesStatesNamesVisitor.h

```diff
@@ -68,6 +70,13 @@ class SwitchesStatesNamesVisitor : public PerfEventVisitor {
       char c);
   static orbit_grpc_protos::ThreadStateSlice::ThreadState GetThreadStateFromBits(uint64_t bits);
 
+  template <typename SchedSwitchPerfEventDataT>
+  void VisitSchedSwitch(uint64_t timestamp, const SchedSwitchPerfEventDataT& event_data,
+                        bool has_switch_out_callstack);
+  template <typename SchedWakeupPerfEventDataT>
+  void VisitSchedWakeup(uint64_t timestamp, const SchedWakeupPerfEventDataT& event_data,
+                        bool has_switch_wakeup_callstack);
```

Comment:
This should be called `has_wakeup_callstack`.

Reply:
Done.